packages/cashier/src/utils/utility.ts

```diff
@@ -0,0 +1,80 @@
+import { getCurrencyDisplayCode } from '@deriv/shared';
+
+class PromiseClass {
+    promise: Promise<any>;
```

Comment:
I think it's better to use unknown instead of any.

Resource: https://wikijs.deriv.cloud/en/Frontend/development-guidelines/ts-guidelines